### DISCUSSION TITLE
Changed eMail Address

### DISCUSSION THREAD

**Discussion by BobUsc** [aclap-dev/video-downloadhelper#454] [Help and Q&A] [answered]
### Your question

How do I change my email address. I changed ISP and theb email address I used to register and pay forthe licence is no longer valid. i.e. it no longer exists.. The Licence is permanently locked as validating so when I click revalidate it sends  validation email to an email address that no longer exists. I have seen posts stating contact @jcfrog but there is no way of contacting this person via email or throughthis site. This is an extremely complex Tecky style site and unfortunately I cant comprehend half ofn what is said

### Addon & CoApp version and details

How do I change my email address. I changed ISP and theb email address I used to register and pay forthe licence is no longer valid. i.e. it no longer exists.. The Licence is permanently locked as validating so when I click revalidate it sends  validation email to an email address that no longer exists. I have seen posts stating contact @jcfrog but there is no way of contacting this person via email or throughthis site. This is an extremely complex Tecky style site and unfortunately I cant comprehend half ofn what is sai

**Comment by jcfrog** [comment, marked as answer]
Hi,
please send us an email at licenses@downloadhelper.net

**Reply by BobUsc** [reply]
Thank you for your fast response

**Comment by GrampaWildWilly** [comment]
Click the link at the top of this page that says Wiki.  That's sort of a user's guide.  You need to get in the habit of referring to that.  You also should get in the habit of searching this forum for answers before you post.

In the Wiki, you will find a kind of Table of Contents down the right side of the page.  One of the chapters has the title "I need help."  That should be a title that catches your attention.  Click it.  Read what you find.  That should help you make progress.  You'll discover that your assumption is false that you can't E-mail the person you need to contact.  Read what's there.  It isn't overly "techy."  You managed to install VDH.  That shows you have some tech skills.  So don't try to convince anybody that you're not "techy" enough to learn from what's here.

**Reply by BobUsc** [reply]
@GrampaWildWilly You puerile little creep!
@GrampaWildWilly This is not an issue that can be resolved in a discussion group. 
It requires direct intervention by jcfrog. The only way to contact him was by posting on this site to get his attention. 
@GrampaWildWilly  I note from your profile you haven't made any posts whatsoever. You just answer other people's posts with inane comments and insults. 
@GrampaWildWilly You are a creepy troll hovering round this site looking for victims to insult. 
@GrampaWildWilly Save your sarcasm and bad manners for someone else. Take your hand out of your pants, get out of your bedroom and get a life. Say hello to mummy and daddy for me when you leave. 
I only used this unnecessarily complex website because it is the only way to get assistance from jcfrog
I don't want to spend hours researching how to use it, as I said I am not a techy geek revelling in complexity
@jcfrog has kindly responded thank you
